.gitignore

```diff
@@ -22,6 +22,9 @@
 # Generated files
 _Version.h
 bdf2u8g
+marlin_config.json
+mczip.h
```

Comment:
Should this be `mc.zip` instead?

Reply:
No. That file is never written outside of `.pio`.